### PR TITLE
Posts fix and update

### PR DESCRIPTION
**Pull request by JonathanPartain**
Include the *'Posts'* section in the *'Home'* Page.

### CODE REVIEW

**Comment by JonathanPartain** [comment]
aims to solve #26

File: _data/theme.yml [hunk 1]
```diff
@@ -3,4 +3,7 @@ skin: night # default, night, plum, sea, soft, steel
 navigation_pages:
   - join_skip.md
   - membership.md
-  - communityposts.md
+  - posts.md
```
Comment (michalspano) [inline _data/theme.yml:6]: Looks good!

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -34,6 +34,7 @@ jobs:
         with:
           source: ./
           destination: ./_site
+          future: true
```
Comment (michalspano) [inline .github/workflows/deploy.yml:37]: What's exactly the purpose of the `future` flag? I see it here and in the **scripts** too.

Comment (michalspano) [reply .github/workflows/deploy.yml:37]: Does it just simply mean that one can set an arbitrary **date** of a post in the future?

Comment (michalspano) [reply .github/workflows/deploy.yml:37]: Resolving this thread, we discussed this **internally**.

**Review by michalspano — COMMENTED** [review]
After an internal discussion, the queries have been addressed by the author of this PR. **LGTM**!

**Pull request merged**